### PR TITLE
add filter configuration in template

### PR DESCRIPTION
**Pull request by hongyh13**
add filter configuration in the template

### CODE REVIEW

File: FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json [hunk 1]
```diff
@@ -53,6 +53,34 @@
...
+            "defaultValue": "",
+            "metadata": {
+                "description": "The resource type you want to export. Will continuous export all resource types if not specified."
+            }
+        },
+        "typeFilters":{
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The type filters you want to filter the export data. Will continuous export all resource types if not specified."
```
Comment (moria97) [inline FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json:81]: Seems `Will continuous export all resource types if not specified` is not suitable for typeFilter field?

Comment (hongyh13) [reply FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json]: Refined.

File: FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json [hunk 2]
```diff
@@ -53,6 +53,34 @@
...
+            "defaultValue": "",
+            "metadata": {
+                "description": "The resources type you want to export. Will continuous export all resource types if not specified."
+            }
+        },
+        "typeFilters":{
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The type filters you want to filter the data. Will continuous export all data if not specified."
```
Comment (moria97) [inline FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json:81]: Maybe we can update the description reference the export doc? https://docs.microsoft.com/en-us/azure/healthcare-apis/fhir/export-data

**Review by moria97 — APPROVED** [review]

File: FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json (hunk 2 above)
Comment (hongyh13) [reply FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json]: thanks for the good suggestion, updated.

**Pull request merged**